gocd/xcdchk.jump.opensuse.yaml

```diff
@@ -0,0 +1,21 @@
+format_version: 3
+pipelines:
+  openSUSE.QA.Maint:
+    group: openSUSE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-xcdchk
+    materials:
+      script:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    timer:
+      spec: 0 */10 * ? * *
```

Comment:
it doesn't make sense to have this script run every 10 minutes - you want this as part of the release pipeline, not as a checker